easybuild/easyconfigs/g/GROMACS/GROMACS-2019.3-foss-2019a.eb

```diff
@@ -0,0 +1,51 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2016 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC,
+#                                 Ghent University / The Francis Crick Institute
+# Authors::
+# * Wiktor Jurkowski <wiktor.jurkowski@gmail.com>
+# * Fotis Georgatos <fotis@cern.ch>
+# * George Tsouloupas <g.tsouloupas@cyi.ac.cy>
+# * Kenneth Hoste <kenneth.hoste@ugent.be>
+# * Adam Huffman <adam.huffman@crick.ac.uk>
+# License::   MIT/GPL
+# 2019.3 version
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+#
+
+name = 'GROMACS'
+version = '2019.3'
+
+homepage = 'http://www.gromacs.org'
+description = """
+GROMACS is a versatile package to perform molecular dynamics,
+ i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.
+"""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = [
+    'https://ftp.gromacs.org/pub/gromacs/',
+    'ftp://ftp.gromacs.org/pub/gromacs/',
+]
+sources = [SOURCELOWER_TAR_GZ]
+patches = [
+    'GROMACS-2019_fix_omp_num_threads_and_google_test_death_style_in_tests.patch',
+    'GROMACS-2019_increase_test_timeout_for_GPU.patch',
```

Comment:
Since this is a non-GPU build, I presume you need to use `GROMACS-2018_dont_do_gpu_tests_on_cpu_build.patch` instead of `GROMACS-2019_increase_test_timeout_for_GPU.patch`

Reply:
@vanzod I have addressed both suggestions. I did not remove the GROMACS-2019_increase_test_timeout_for_GPU.patch as it was working for me without any problems. I did not see the second comment after I uploaded the first patched build file.